experiments/tf_trainer/common/tfrecord_simple_test.py

```diff
@@ -46,44 +49,35 @@ def setUp(self):
     self.ex_tensor = tf.convert_to_tensor(ex.SerializeToString(), dtype=tf.string)
 
   def test_TFSimpleRecordInput_unrounded(self):
-    dataset_input = tfrecord_simple.TFSimpleRecordInput(
-        train_path=None,
-        validate_path=None,
-        text_feature="comment",
-        labels={"label": tf.float32},
-        round_labels=False)
+    FLAGS.round_labels = False
+    FLAGS.labels = 'label'
+    dataset_input = tfrecord_simple.TFSimpleRecordInput()
 
     with self.test_session():
       features, labels = dataset_input._read_tf_example(self.ex_tensor)
-      self.assertEqual(features["comment"].eval(), b"Hi there Bob")
+      self.assertEqual(features["text"].eval(), b"Hi there Bob")
```

Comment:
why not use single quotes like everywhere else?

Reply:
This is not consistent in our code, but doing it.

Reply:
Urg... you're right. OK, we'll work our way to cleaning this up.